guides/source/active_record_migrations.md

```diff
@@ -809,7 +809,7 @@ Active Record believes that it has already been run.
 
 ### Running Migrations in Different Environments
 
-By default running `rake db:migrate` will run in the `development` environment.
+By default running `rake or bin/rails db:migrate` will run in the `development` environment.
```

Comment:
@maclover7 yup !! here is my intentaion to show the various changes. If community agree to change all `rake` implemented rails commands using `bin/rails` in guides or elsewhere. i'll change all over the places. More over can directly push into docrails.